src/Components/FeelingRoom/anxiousAnimations.scss

```diff
@@ -0,0 +1,218 @@
+.anxiousRoom {
```

Comment:
This is a heafty chunk og code that I'm excited tot alk through on our next call. The end result is beautiful though :)